apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -382,6 +414,15 @@ export default function teacherSections(state=initialState, action) {
     };
   }
 
+  if (action.type === SET_DEFAULT_ASSIGNMENT) {
+    const { courseId, scriptId } = action;
+    return {
+      ...state,
+      defaultCourseId: courseId,
+      defaultScriptId: scriptId
+    };
```

Comment:
I'm confused - don't we want to launch straight into the new section flow when we pass these queryparams?  Why wouldn't we dispatch _that_ and set the course and script ids on the `sectionBeingEdited` instead of introducing some new and independent state?